lib/internal/http2/util.js

```diff
@@ -484,6 +485,9 @@ function mapToHeaders(map,
       count++;
       continue;
     }
+    if (key.indexOf(' ') > 0) {
```

Comment:
Shouldn't this be `!= 0` or `>= 0` to handle space at the start of a name?

Reply:
Done.